conans/client/tools/system_pm.py

```diff
@@ -114,6 +116,18 @@ def install(self, packages, update=True, force=False):
             self.update()
         self._install_any(packages)
 
+    def _parse_packages_arch(self, packages, platforms):
+        if self._settings and cross_building(self._settings):
+            _, build_arch, _, host_arch = get_cross_building_settings(self._settings)
+            arch = host_arch or build_arch
+            parsed_packages = []
+            for package in packages:
+                if isinstance(package, str):
+                    for package_name in package.split(" "):
+                        parsed_packages.append(self._tool._parse_package(package_name, arch, platforms))
+            return parsed_packages
```

Comment:
we might end up returning an empty list here, is it expected?

Reply:
I think yes. If we can't parse something, because we don't know all possible suffixes and prefixes, then we return an empty list, otherwise we would need raise an error. The use is able to customize the suffix/prefix list, so I think breaking is not a good behavior for this case.

Reply:
done